src/pages/reader/epub.vue

```diff
@@ -129,7 +129,10 @@ export default {
       });
       this.rendition.display();
       this.rendition.on('rendered', (cfiRange, contents) => {
-        this.selectedChapter = this.rendition.currentLocation().start.href;
+        const path = this.rendition.currentLocation().start.href;
+        const pathArr = path.split('/');
+        this.selectedChapter = pathArr.pop();
+        this.dirPath = pathArr.join('/');
```

Comment:
`dirPath` should be defined in data, and maybe cleared on reload or unmount

Reply:
missed one line when committing...
`dirPath` and `selectedChapter` will be updated when rendering